source/agora/consensus/validation/Block.d

```diff
@@ -693,20 +693,6 @@ version (unittest)
                 file, line, block.header.height, reason);
         return reason is null;
     }
-
-    /// Ditto but returns `bool`, only usable in unittests
-    public bool isNotValid (in Block block, Height prev_height,
-        Hash prev_hash, scope UTXOFinder findUTXO,
-        size_t active_enrollments, size_t enrolled_validators, scope FeeChecker checkFee,
-        scope EnrollmentFinder findEnrollment, Hash random_seed = Hash.init,
-        ulong enrollment_cycle = 0, ulong prev_time_offset = 0, ulong curr_time_offset = ulong.max,
-        Duration block_time_tolerance = 100.seconds) nothrow @safe
-    {
-        return isValidcheck(block, prev_height, prev_hash, findUTXO,
```

Comment:
This one is dong the check but not logging. `isValid` will log if it is not. See comment in line 678.